src/shared/components/event-news-meta/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import styles from '../events-list/style.css';
+
+import TagsList from '../tags-list';
+import EventLinksList from '../event-links-list';
+
+const EventNewsMeta = ({
+  internalLinks,
+  externalLinks,
+  tags,
+}) => {
+  const linksSection = (<div>
+    {
+      externalLinks || internalLinks ?
+        <div className={styles.eventLinks}>
+          <EventLinksList
+            linkList={externalLinks}
+            listType="external" />
+          <EventLinksList
+            linkList={internalLinks}
+            listType="internal" />
+        </div>
+      : <noscript />
```

Comment:
The issue I have with extracted functions is that it's very hard to see the whole markup at the same time - you have to jump around the code to assemble it in your head. Ternary is also not great, but the only real option (in my super personal opinion) is not to use JSX.